Packs/DeprecatedContent/Playbooks/playbook-Account_Enrichment_-_Generic_README.md

```diff
@@ -1,42 +1,42 @@
-DEPRECATED. Use "Account Enrichment - Generic v2.1" playbook instead. Enriches accounts using one or more integration.
+Deprecated. Use "Account Enrichment - Generic v2.1" playbook instead. Enrich Accounts using one or more integrations
 
 ## Dependencies
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
-## Sub-playbooks
+### Sub-playbooks
 This playbook does not use any sub-playbooks.
 
-## Integrations
-This playbook has no integrations.
+### Integrations
+This playbook does not use any integrations.
 
-## Scripts
-* ADGetUser
+### Scripts
 * Exists
+* ADGetUser
 
-## Commands
+### Commands
 * ad-get-user
 
 ## Playbook Inputs
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
-| --- | --- | --- | --- | 
-| Username | The username to enrich. | ${Account.Username} |Optional |
+| --- | --- | --- | --- |
+| Username | The Username to enrich | ${Account.Username} | Optional |
 
 ## Playbook Outputs
 ---
 
 | **Path** | **Description** | **Type** |
 | --- | --- | --- |
-| Account | The account object. | unknown |
-| Account.Type | The type of the account entity. | string |
-| Account.ID | The unique account DN (Distinguished Name). | string |
-| Account.Username | The account username. | string |
-| Account.Email | The email address associated with the account. | unknown |
-| Account.Groups | The groups the account is part of. | unknown |
-| Account.DisplayName | The account display name. | string |
-| Account.Manager | The account's manager. | string |
+| Account | Account object | unknown |
+| Account.Type | Type of the Account entity | string |
+| Account.ID | The unique Account DN \(Distinguished Name\) | string |
+| Account.Username | The Account username | string |
+| Account.Email | The email address associated with the Account | unknown |
+| Account.Groups | The groups the Account is part of | unknown |
+| Account.DisplayName | The Account display name | string |
+| Account.Manager | The Account's manager | string |
 
 ## Playbook Image
 ---
-![Account_Enrichment_Generic](https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Account_Enrichment_Generic.png)
+![Account Enrichment - Generic](Insert the link to your image here)
```

Comment:
@tomer-pan you deleted the image

Reply:
@David-BMS Done (including all of the other md files which the image was deleted in)